python/lsst/ip/isr/isrTask.py

```diff
@@ -60,7 +60,7 @@
 class IsrTaskConnections(pipeBase.PipelineTaskConnections,
                          dimensions={"instrument", "visit", "detector"},
                          defaultTemplates={}):
-    ccdExposure = cT.PrerequisiteInput(
+    ccdExposure = cT.Input(
```

Comment:
Is this change related to DM-23702? Should we be making that change here or deferring to that ticket?

Reply:
I am currently implementing this change on DM-23702.

Reply:
Yes, I put this in here to help with the pipeline testing in DM-2361. I'll take it out.